app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, origin, copy = command.split()
+
+    if origin == copy or cp == "cp":
+        return None
```

Comment:
wrong condition here, we should use cp command
also no need to use return None here, u can just use backward condition for code below